.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
It can be helpful to be explicit with what is text and the correct line endings.

*.js text eol=lf
*.json text eol=lf
*.md text eol=lf
...